index.html

```diff
@@ -90,19 +108,12 @@
       }
 
       /* Re-enable pointer events for specific interactive elements */
-      .contentContainer a,
-      .contentContainer button,
-      .contentContainer input,
-      .contentContainer select,
-      .contentContainer textarea,
-      .contentContainer label,
+      .contentContainer > *
```

Comment:
changed the target here, assume that you wanted like all children which you can use `> *` for